ecosystem/sep-0010.md

```diff
@@ -18,12 +18,21 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
-The authentication flow is as follows:
+It involves three components:
+- A domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` and `SIGNING_KEY`, referred to as the home domain.
+- An endpoint providing the GET and POST operations discussed in this document.
+- A client who will authenticate using a Stellar account.
+
+The discovery flow is as follows:
+
+1. The client retrieves the [SEP-1 stellar.toml](sep-0001.md) from the home domain. The home domain hosts the stellar.toml.
+1. The client looks up the `WEB_AUTH_ENDPOINT` and `SIGNING_KEY` from the [SEP-1 stellar.toml](sep-0001.md).
 
-1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction
+The authentication flow is as follows:
+1. The client obtains a unique [`challenge`](#challenge) from the `WEB_AUTH_ENDPOINT`, which is represented as specially formed Stellar transaction
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
-1. The client verifies that the transaction is signed by the `SIGNING_KEY` specified by the requested service's [SEP-1 stellar.toml](sep-0001.md).
-1. The client verifies that the transaction has a single Manage Data operation with its source account set to the user's account and value set to a nonce value. The client ignores the home domain included.
+1. The client verifies that the transaction is signed by the `SIGNING_KEY` obtained through discovery flow.
+1. The client verifies that the transaction has a single Manage Data operation with its source account set to the user's account and value set to a nonce value. The client verifies the home domain included is the home domain from the discovery flow.
```

Comment:
I think this second sentence should be it's own bullet for readability.

Reply:
👍  1d066da